lib/matplotlib/cbook.py

```diff
@@ -2344,7 +2343,7 @@ def pts_to_poststep(x, *args):
     # do normalization
     vertices = _step_validation(x, *args)
     # create the output array
-    steps = ma.zeros((vertices.shape[0], 2 * len(x) - 1), np.float)
+    steps = np.zeros((vertices.shape[0], 2 * len(x) - 1), np.float)
```

Comment:
There's quite a few `np.float` left; this one is safe to replace but I'd need to do some more research to figure out how this changes calls to `np.issubdtype` and others.